src/Providers/DataGrid/Wijmo/Grid/FlexGrid.ts

```diff
@@ -96,7 +168,14 @@ namespace Providers.DataGrid.Wijmo.Grid {
 		public addColumn(col: OSFramework.DataGrid.Column.IColumn): Promise<void> {
 			super.addColumn(col);
 
+			//TODO: Workaround to be removed with wijmo update (2015v1?). See task ROU-11338.
+			this._useWorkaroundRou11338 =
+				this._useWorkaroundRou11338 || col.columnType === OSFramework.DataGrid.Enum.ColumnType.Dropdown;
+
 			if (this.isReady) {
+				//TODO: Workaround to be removed with wijmo update (2015v1?). See task ROU-11338.
+				this._dropdownWorkaround();
```

Comment:
Why is it necessary to call the `this._dropdownWorkaround` here and during the build?

Reply:
There are 2 situations:
- when the grid is built having a dropdown column already
- when the grid is built without a dropdown column, and a dropdown column is added afterwards

The idea of this approach is to avoid adding the handlers when no dropdown column is used.